lib/route.js

```diff
@@ -970,6 +980,9 @@ function getAssociations(schema) {
     return associations;
 }
 
+route.getRestMethods = function () {
```

Comment:
I've already limited number of places this list was used by introducing this method.
Initially I also wanted to have that list auto populated, but couldn't find clear way for that.